PE/carve.py

```diff
@@ -39,17 +39,17 @@ def carve(pbytes, offset=0):
 
         newoff = struct.unpack('<I', xorstatic( pbytes[e_lfanew : e_lfanew + 4], i))[0]
 
+        nextres = pbytes.find(mzx, off+1)
+        if nextres != -1:
+            todo.append( (nextres, mzx, pex, i) )
+
```

Comment:
can we get a small unittest to prove the correct behavior here?

Reply:
Ya. Lemme see if I can scrounge up the binary I was using.